source/general/pkteam.rst

```diff
@@ -71,9 +72,11 @@ Our administrators are the people in charge of organizing the entire team and co
 
 +++++
 
+.. _jointheteam:
+
 Join the Team
 =============
-The ProjectKorra staff is always looking for qualified people to join its team. Whether your passion lies in staying up late and writing lines of code, graphic design, video production, and helping administrate Minecraft servers, there’s a place for you on our team.
+The ProjectKorra team is always looking for qualified people to join its team. Whether your passion lies in staying up late and writing lines of code, graphic design, video production, or helping administrate Minecraft servers, there’s a place for you on our team.
```

Comment:
"Whether your passion lies in writing lines of code, graphic design/video production, or helping run Minecraft servers, there's a place for you on our team"

Reply:
what's your point

Reply:
Omit the "in staying up late part"

Reply:
That's what we do tho?